osu.Game.Rulesets.Osu/Mods/OsuModWiggle.cs

```diff
@@ -22,8 +24,21 @@ internal class OsuModWiggle : ModWithVisibilityAdjustment
         public override double ScoreMultiplier => 1;
         public override Type[] IncompatibleMods => new[] { typeof(OsuModTransform) };
 
-        private const int wiggle_duration = 90; // (ms) Higher = fewer wiggles
-        private const int wiggle_strength = 10; // Higher = stronger wiggles
+        [SettingSource("Strength")]
+        public BindableDouble WiggleStrength { get; } = new BindableDouble(10)
+        {
+            MinValue = 1f,
+            MaxValue = 15f,
+            Precision = .5f
+        };
+
```

Comment:
Would probably be better to have the setting range as multiples of default value if the unit of wiggle strength is arbitrary. e.g. from 0.1x to 2x

Reply:
done